src/Akihabara.Tests/Framework/Packet/EglSurfaceHolderPacketTest.cs

```diff
@@ -0,0 +1,92 @@
+﻿using Akihabara.Core;
+using Akihabara.Framework;
+using Akihabara.Framework.Packet;
+using Akihabara.Framework.Port;
+using Akihabara.Gpu;
+using NUnit.Framework;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Akihabara.Tests.Framework.Packet
+{
+#if OPENGL_ES
+    [TestFixture]
+    public class EglSurfaceHolderPacketTest
+    {
+#region Constructor

```

Comment:
Quick styling inconsistency: some `#region`s start at the beginning of the line, while some start at some level of indentation.

Reply:
I'll try fixing it now, and if i can't, I'll do that when i come home